src/jdk.javadoc/share/classes/jdk/javadoc/internal/html/HtmlTree.java

```diff
@@ -365,6 +375,44 @@ public static HtmlTree A(URI ref, Content body) {
                 .add(body);
     }
 
+    /**
+     * Creates an HTML {@code BODY} element with the given style.
+     *
+     * @param style the style
+     * @return the element
+     */
+    public static HtmlTree BODY(HtmlStyle style) {
+        return new HtmlTree(HtmlTag.BODY)
+                .setStyle(style);
+    }
+
+    private static HtmlTree BR_INSTANCE = unmodifiableTree(HtmlTag.BR);
+
+    /**
+     * {@return an HTML {@code BR} element}
+     */
+    public static HtmlTree BR() {
+        return BR_INSTANCE;
+    }
+
+    /**
+     * Creates an HTML {@code BUTTON} element with the given id.
+     *
+     * @return the element
```

Comment:
Missing `@param` for `id`.

Reply:
Given the possibility of exporting this package to other JDK modules, it might be interesting to add a test that runs DocLint on this package.